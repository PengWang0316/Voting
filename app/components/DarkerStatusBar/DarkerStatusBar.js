import React from 'react';
import { StatusBar } from 'react-native';

import Theme from '../../Theme';

export const DarkerStatusBar = () => (
  <StatusBar
    barStyle="light-content"
    backgroundColor={Theme.primary.dark}
  />);
export default DarkerStatusBar;
