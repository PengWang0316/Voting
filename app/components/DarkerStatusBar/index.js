import DarkerStatusBar from './DarkerStatusBar';

export default DarkerStatusBar;
