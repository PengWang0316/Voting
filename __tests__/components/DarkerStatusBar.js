import React from 'react';
import renderer from 'react-test-renderer';

import { DarkerStatusBar } from '../../app/components/DarkerStatusBar/DarkerStatusBar';

// jest.mock('react-native', () => ({ StatusBar: 'StatusBar' }));
jest.mock('StatusBar', () => 'StatusBar');

describe('DarkerStatusBar Test', () => {
  test('snapshot', () => expect(renderer.create(<DarkerStatusBar />).toJSON()).toMatchSnapshot());
});
