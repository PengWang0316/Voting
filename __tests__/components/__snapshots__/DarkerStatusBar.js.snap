// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DarkerStatusBar Test snapshot 1`] = `
<StatusBar
  backgroundColor="#3e6e85"
  barStyle="light-content"
/>
`;
